api-client/src/stringifyFormData.ts

```diff
@@ -1,14 +1,14 @@
 // stringifies FormData from react-native or the browser variant of it
 export function stringifyFormData(formData: any): string {
-  const entries = {}
+  const entries: Record<string, any> = {}
 
   if (formData?.getParts) {
     for (const part of formData.getParts()) {
-      entries[part.fieldName] = part.string
+      entries[part.fieldName as string] = part.string
     }
   } else {
     for (const [key, value] of formData.entries()) {
-      entries[key] = value
+      entries[key as string] = value
```

Comment:
We probably should add tests for this. Especially as the types are a little bit hacky.

Reply:
This is only used for showing a nice error, so I don't think this is critical. Maybe we could just use `JSON.stringify()` as well instead of this?

Reply:
Sure, it is not critical. Imo just JSON.stringify would be better, especially as we just print this message to the console right?

Reply:
Replaced with JSON.stringify()